prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1098,6 +1101,65 @@ static int rxr_ep_set_cuda_api_permitted(struct rxr_ep *ep, bool cuda_api_permit
 	return 0;
 }
 
+/**
+ * @brief set use_device_rdma flag in rxr_ep
+ * @param[in,out]	ep			endpoint
+ * @param[in]		use_device_rdma		when true, use device RDMA capabilities.
+ * @return		0 on success
+ *
+ * @related rxr_ep
+ */
+static int rxr_ep_set_use_device_rdma(struct rxr_ep *ep, bool use_device_rdma)
+{
+	uint32_t api_version =
+		 rxr_ep_domain(ep)->util_domain.fabric->fabric_fid.api_version;
+
+	int env_setting = rxr_env_get_use_device_rdma();
```

Comment:
Following previous comments. It seems we can use the combination of `rxr_get_env_default_use_device_rdma()` and `rxr_env_has_use_device_rdma()`

Reply:
yes.  We could use a single function which either returns -1, 0, or 1, or we could use two functions.  You prefer two functions?

Reply:
Yes. I prefer two functions, both of them the return a boolean.

I do not like a function's return can be both a return code and a value,

Reply:
Done.